.github/workflows/check-components-changelog.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         name: Check CHANGELOG diff
         runs-on: ubuntu-latest
         permissions:
-          contents: read
+            contents: read
```

Comment:
My editor did the same correction yesterday, which I had to revert manually. Only happened once, so not sure what the problem is.

Reply:
I believe this is the result of `wp-scripts format` being run through `lint-staged`. Perhaps `@wordpress/prettier-config` is enforcing such rules?